### PR TITLE
Np 1726/refactor unstyled table view changes fix selected columns

### PR DESCRIPTION
**Pull request by olhabn**
FObject or Reference property displayed as 'Selected' even if no child property is selected

@jlhughes @kgrgreer please review

### CODE REVIEW

File: src/foam/u2/view/ColumnConfigView.js [hunk 1]
```diff
@@ -726,8 +713,16 @@ foam.CLASS({
...
           }
         }
-        return arr;
+        if ( arr && arr.length > 0 )
+          return arr;
       }
+      if ( this.level === 0 ) {
+        if ( foam.Array.isInstance(this.rootProperty) )
+          return [[this.rootProperty[0]]];
+        else
```
Comment (kgrgreer) [inline src/foam/u2/view/ColumnConfigView.js]: No need for 'else' if you've already returned inn the 'if'.

File: src/foam/u2/view/ColumnConfigView.js [hunk 2]
```diff
@@ -726,8 +713,16 @@ foam.CLASS({
...
+          return [this.rootProperty];
+      } else
```
Comment (kgrgreer) [inline src/foam/u2/view/ColumnConfigView.js]: No need for 'else' if you've already returned inn the 'if'.

File: src/foam/u2/view/UnstyledTableView.js [hunk 3]
```diff
@@ -293,6 +281,7 @@ foam.CLASS({
 
             return this.E().
               addClass(view.myClass('tr')).
+              style({ 'min-width': this.tableWidth_$ }).
```
Comment (kgrgreer) [inline src/foam/u2/view/UnstyledTableView.js:284]: Does tableWidth_ ever change its value?

Comment (olhabn) [reply src/foam/u2/view/UnstyledTableView.js:284]: it should be on column selection change as it's the sum of the columns tableWidth

File: src/foam/u2/view/ColumnConfigView.js (hunk 1 above)
Comment (olhabn) [reply src/foam/u2/view/ColumnConfigView.js]: fixed

File: src/foam/u2/view/ColumnConfigView.js (hunk 2 above)
Comment (olhabn) [reply src/foam/u2/view/ColumnConfigView.js]: fixed

**Pull request merged**